README.md

```diff
@@ -25,6 +25,9 @@ are removed by this plugin, so this option gives you a chance to "exclude them f
 * :warning: Make sure you enter a string 'a-string' instead of JS regexp like `/a-string/`, Gatsby has trouble handing off regular expressions to plugins. The string you pass in
 will be handled as a regular expression for you. 
 
+`excludePaths`: `string[] (optional)`
```

Comment:
Could this be an array of regex? I think that there are some cases where a `includes` is not going to be flexible enough.